homeassistant/components/tessie/sensor.py

```diff
@@ -222,4 +222,4 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the state of the sensor."""
-        return self.entity_description.value_fn(self.value)
+        return self.entity_description.value_fn(self.coordinator.data[self.key])
```

Comment:
It looks like some sensor changes accidentally made it in with merges

Reply:
self.value got removed in this PR.
I have pulled this change out into https://github.com/home-assistant/core/pull/105838

Reply:
Thanks, this PR doesnt touch sensors anymore.